net/ghttp/ghttp_server_service_handler.go

```diff
@@ -169,7 +177,26 @@ func (s *Server) checkAndCreateFuncInfo(f interface{}, pkgPath, structName, meth
 			err = gerror.NewCodef(
 				gcode.CodeInvalidParameter,
 				`invalid handler: defined as "%s", but the first input parameter should be type of "context.Context"`,
-				reflect.TypeOf(f).String(),
+				reflectType.String(),
+			)
+			return
+		}
+
+		if reflectType.In(1).Kind() != reflect.Ptr ||
+			(reflectType.In(1).Kind() == reflect.Ptr && reflectType.In(1).Elem().Kind() != reflect.Struct) {
+			err = gerror.NewCodef(
+				gcode.CodeInvalidParameter,
+				`invalid handler: defined as "%s", but the second input parameter should be type of pointer to struct(*BizReq)`,
```

Comment:
`invalid handler: defined as "%s", but the second input parameter should be type of pointer to struct like "*BizReq"`